pkg/domain/domain.go

```diff
@@ -1834,16 +1848,21 @@ func (do *Domain) LoadPrivilegeLoop(sctx sessionctx.Context) error {
 
 		var count int
 		for {
+			var userList []string
+			var resp clientv3.WatchResponse
 			ok := true
 			select {
 			case <-do.exit:
 				return
-			case _, ok = <-watchCh:
+			case resp, ok = <-watchCh:
+				if ok {
```

Comment:
Though it is not introduced in this PR, do we need to handle the `ok == false` here? If the watchCh closed unexpectedly for some reason, this loop would run frequent.

Reply:
Just like this issue #49273

Reply:
line 1863 will recreate the watch channel...  if it's not domain close.

Reply:
There is a sleep 1s ....